tests/integration/test_inspect_table.py

```diff
@@ -672,126 +672,141 @@ def test_inspect_files(
     # append more data
     tbl.append(arrow_table_with_null)
 
-    df = tbl.refresh().inspect.files()
+    # configure table properties
+    if format_version == 2:
+        with tbl.transaction() as txn:
+            txn.set_properties({"write.delete.mode": "merge-on-read"})
+    spark.sql(f"DELETE FROM {identifier} WHERE int = 1")
```

Comment:
is this to produce delete files?

Reply:
Yes

Reply:
About this: https://github.com/apache/iceberg-python/pull/1066#discussion_r1740146920
I have checked in Spark shell as well, the values are `null`

Reply:
do you know if this produces positional deletes, equality deletes, or both?

Reply:
I think Spark only produces positional delete files. Flink might produce equality delete files.

Reply:
hm, good point. I can't find any reference with Spark and equality delete

Reply:
Spark streaming might produce equality deletes, I tried with Flink, created an Iceberg table and upserted some data into it, observed that there were both positional and equality deletes files, which looked weird to me.

Reply:
this was a nit comment btw, not blocking. we don't necessary have to test for equality delete files here

Reply:
Alright, other than this, other comments are resolved.

Reply:
> Spark only produces positional delete files. Flink might produce equality delete files.

That's my understanding as well. I think just testing for positional deletes is okay for now, and maybe we can think of ways in adding equality deletes in our test suite in the future.